### ISSUE TITLE
Ansible won't install

### ISSUE THREAD

**Issue by walterdolce** [neillturner/kitchen-ansible#237]
Hi, 
I'm trying to install Ansible with the following `.kitchen.yml` configuration but it fails.

```
---
driver:
  name: vagrant
  provider: virtualbox

provisioner:
  name: ansible_playbook
  hosts: all
  ansible_vault_password_file: ./vault_passphrase_development
  ansible_verbosity: 4
  ansible_verbose: true
  require_ansible_repo: false
  require_chef_for_busser: false
  require_ruby_for_busser: false
  require_pip: true
  requirements_path: ./requirements.yml
  roles_path: ./roles

verifier:
  name: inspec

platforms:
  - name: debian
    driver:
      box: debian/jessie64
      box_version: 8.7.0

suites:
  - name: default
    provisioner:
      playbook: ./development-app-playbook.yml
```

Error message:
```
Collecting ansible==
         Could not find a version that satisfies the requirement ansible== (from versions: 1.0, 1.1, 1.2, 1.2.1, 1.2.2, 1.2.3, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.4, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.5, 1.5, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5, 1.6, 1.6.1, 1.6.2, 1.6.3, 1.6.4, 1.6.5, 1.6.6, 1.6.7, 1.6.8, 1.6.9, 1.6.10, 1.7, 1.7.1, 1.7.2, 1.8, 1.8.1, 1.8.2, 1.8.3, 1.8.4, 1.9.0.1, 1.9.1, 1.9.2, 1.9.3, 1.9.4, 1.9.5, 1.9.6, 2.0.0.0, 2.0.0.1, 2.0.0.2, 2.0.1.0, 2.0.2.0, 2.1.0.0, 2.1.1.0, 2.1.2.0, 2.1.3.0, 2.1.4.0, 2.1.5.0, 2.2.0.0, 2.2.1.0, 2.2.2.0, 2.3.0.0)
       No matching distribution found for ansible==
```

Looks like it's like it's waiting for a version to be passed but none is given. From the docs I don't understand whether there's anything missing..

THank you

**Comment by neillturner** [comment]
its a bug workaround is add parameter 
ansible_version: latest

**Comment by walterdolce** [comment]
@neillturner thanks, the workaround worked. Will leave this open until this is retrofitted into the provisioner itself.

**Comment by neillturner** [comment]
i will write a fix on weekend and release. thanks for taking the time to create issue.

**Comment by neillturner** [comment]
fixed version 0.46.6
